### PR TITLE
docs: mixins.md

### PR DESCRIPTION
**Pull request by watonyweng**
⚠️⚠️⚠️ DISCLAIMER ⚠️⚠️⚠️:
Thanks for your interest in submitting a PR! At this time, because the docs are in beta, the team is currently in the midst of changes. As a result, we are not ready for additional contributions yet.

How can I provide feedback?
To be respectful of your time and ideas, please create an issue instead of a pull request. Otherwise, we will have to simply close your pull request at this time.

Thanks for your understanding! 
#317

### CODE REVIEW

**Comment by veaba** [comment]
@Jinjiang @Justineo 现在基本对 mixin 是保留原文，但 `v2` 迁移过来的 wiki 是要翻译为 “混入”，这样是否恰当？或者有更改的必要？

**Comment by Justineo** [comment]
我感觉可以保留 mixin 不翻译，看看勾股的意见。

@Jinjiang

File: src/guide/mixins.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ app.mount('#mixins-global')
 
 ## 自定义选项合并策略
 
-自定义选项将使用默认策略，即简单地覆盖已有值。如果想让自定义选项以自定义逻辑合并，可以向 `app.config.optionMergeStrategies` 添加一个函数：
+自定义选项将使用默认策略，即简单地覆盖已有值。如果想让自定义选项进行自定义逻辑合并，可以向 `app.config.optionMergeStrategies` 添加一个函数：
```
Comment (Justineo) [inline src/guide/mixins.md:167]:
```suggestion
自定义选项在合并时，默认策略为简单地覆盖已有值。如果想让某个自定义选项以自定义逻辑进行合并，可以在 `app.config.optionMergeStrategies` 中添加一个函数：
```

**Review by Justineo — CHANGES_REQUESTED** [review]

**Comment by Jinjiang** [comment]
认同。我们统一保留 mixin 吧，同时在 wiki 里注明一下这个词正常情况下是不翻译的。谢谢

**Pull request merged**